README.md

```diff
@@ -37,14 +37,11 @@ You can install this program globally with `npm i -g zksync-cli` or run the comm
 
 - `zksync-cli --version`: Returns the current version
 
-- `zksync-cli localnet`: Manages a local zkSync Era and Ethereum L1 testnet (it requires docker running on your system). It supports a set of sub-subcommands:
-  - `zksync-cli localnet up`: Bootstrap L1 and L2 localnets.
-  - `zksync-cli localnet down`: clear L1 and L2 localnets.
-  - `zksync-cli localnet start`: start L1 and L2 localnets.
-  - `zksync-cli localnet stop`: stop L1 and L2 localnets.
-  - `zksync-cli localnet logs`: Display logs.
-  - `zksync-cli localnet help`: Display this message and quit.
-  - `zksync-cli localnet wallets`: Display seeded wallet keys.
+- `zksync-cli local` - All-in-one tool for local zkSync development. It supports a set of sub-subcommands:
+  - `zksync-cli local start` - start local setup (will ask to configure if starting for the first time)
```

Comment:
For the first time configuration, I selected In Memory node and then it asked if I wanted Block explorer and / or Portal. I don't think the block explorer is supported yet for in-memory node? We should update to remove block explorer as an optional service if the user selects in-memory node for the time being. Wdyt?

Reply:
It is fine because it is not merged to main and don't plan to announce it yet